swagger-spec/swagger.yaml

```
@@ -1379,3 +1456,37 @@ paths:
 
   /provider/collections/{collection_id}/moderators/{moderator_id}/:
     $ref: 'collection_provider/moderators_detail.yaml'
+
+  ################################
+  #   NOTIFICATION SUBSCRIPTIONS #
+  ################################
+
+  /subscriptions/:
+    $ref: 'subscriptions/subscription_list.yaml'
+
+  /subscriptions/{subscription_id}/:
+    $ref: 'subscriptions/subscription_detail.yaml'
+
+  /providers/collections/{provider_id}/subscriptions/:
+    $ref: 'subscriptions/collection_provider_subscription_list.yaml'
+
+  /providers/collections/{provider_id}/subscriptions/{subscription_id}/:
+    $ref: 'subscriptions/collection_provider_subscription_detail.yaml'
+
+  /providers/preprints/{provider_id}/subscriptions/:
+    $ref: 'subscriptions/preprint_provider_subscription_list.yaml'
+
+  /providers/preprints/{provider_id}/subscriptions/{subscription_id}/:
+    $ref: 'subscriptions/preprint_provider_subscription_detail.yaml'
+
+  /providers/registrations/{provider_id}/subscriptions/:
+    $ref: 'subscriptions/registration_provider_subscription_list.yaml'
+
+  /providers/registrations/{provider_id}/subscriptions/{subscription_id}/:
+    $ref: 'subscriptions/registration_provider_subscription_detail.yaml'
+
+  /registration_subscriptions/{subscription_id}/:
+    $ref: 'subscriptions/registration_subscription_detail.yaml'
+
+  /oollection_subscriptions/{subscription_id}/:
```

Looks like a typo, it should be /collection_subscriptions/{subscription_id}
```suggestion
  /collection_subscriptions/{subscription_id}/:
```